src/pages/BirthdayPicker.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-restricted-syntax */
+import React, { useState } from 'react';
+import {
+  Theme,
+  makeStyles,
+  Grid,
+} from '@material-ui/core';
+
+import DOBPicker from '../components/DOBPicker';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  home: {
+    marginTop: theme.spacing(1),
+  },
+}));
+
+type BirthdayPickerType = {
+
+};
+
+const BirthdayPicker: React.FC<BirthdayPickerType> = () => {
+  const classes = useStyles();
+  const [dob1, setDob1] = useState<Date>();
+  const [dob2, setDob2] = useState<Date>(new Date(2000, 0, 1));
+  const [dob3, setDob3] = useState<Date>();
+  const [formJson, setFormJson] = useState({});
+  return (
+    <form onSubmit={(e) => {
+      e.stopPropagation();
+      e.preventDefault();
+      const inputs = (e.target as HTMLFormElement).querySelectorAll('input');
```

Comment:
Here, when I used `e: React.FormEvent<HTMLFormElement>` I was having issues with `querySelectorAll` on `e.target`:
![image](https://user-images.githubusercontent.com/6208180/108611647-89e63000-73ae-11eb-894b-0ecb19df4776.png)
![image](https://user-images.githubusercontent.com/6208180/108611679-cd409e80-73ae-11eb-922d-db4a6549df7a.png)

Any ideas on how to resolve that?

Reply:
I only know this because I learned it like two days ago in a completely unrelated situation. But you can do `e.currentTarget`, which comes from the React.SyntheticEvent class. 

<img width="916" alt="image" src="https://user-images.githubusercontent.com/76490513/108661411-a0f55280-7480-11eb-9044-8494f81a4831.png">